packages/auth/cloud/cloudformation.spec.ts

```diff
@@ -0,0 +1,3 @@
+test('test 1', () => {
```

Comment:
Fix tests.

Reply:
How do I do that? 

expect(1).toBe(1)?

Reply:
No. You should test the method `getTemplate`.

```
expect(getTemplate())...
```

Reply:
Done.